frontend/components/FormLayout.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <div class="formlayout">
+        <slot></slot>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'FormLayout',
+    data () {
+        return {}
+    },
```

Comment:
nit: blank `data()` can be removed
```suggestion
    name: 'FormLayout'
```